addons/stock/tests/test_packing.py

```diff
@@ -1975,3 +1975,32 @@ def test_pack_in_pack_already_packed(self):
         self.assertEqual(boxes.package_dest_id.picking_ids, receipt)
         self.assertFalse((pallet | container).package_dest_id)
         self.assertFalse((pallet | container).picking_ids)
+
+    def test_package_propagation_different_uom(self):
+        supplier_location = self.env.ref('stock.stock_location_suppliers')
+        self.warehouse.in_type_id.set_package_type = False
+        self.warehouse.reception_steps = "two_steps"
+        pack_of_6 = self.ref('uom.product_uom_pack_6')
+        self.productA.uom_ids = self.env['uom.uom'].browse(pack_of_6)
+        self.env['stock.quant']._update_available_quantity(self.productA, self.stock_location, 30.0)
```

Comment:
```suggestion
```
Why do you need quantities in `WH/Stock` if you're checking a two-step input flow?